### PR TITLE
Chinese Translation

### PR DESCRIPTION
**Pull request by billchenchina**
Co-authored-by: Cyanoxygen <cyanoxygen2725@gmail.com>

### CODE REVIEW

File: zh-cn/index.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>不要为了提问而提问</title>
+  <meta property="og:title" content="Don't ask to ask, just ask" />
```
Comment (tulir) [inline zh-cn/index.html:5]: This should also be translated

File: zh-cn/index.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
```
Comment (tulir) [inline zh-cn/index.html:7]:
```suggestion
  <meta property="og:url" content="https://dontasktoask.com/zh-cn" />
```

File: zh-cn/index.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        display: table;
+      }
+      blockquote > * {
+        display: table-cell;
+      }
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
```
Comment (tulir) [inline zh-cn/index.html:81]: The CSS was recently moved to a separate file to reduce duplication

```suggestion
  <link rel="stylesheet" type="text/css" href="/style.css">
```

File: zh-cn/index.html [hunk 4]
```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>不要为了提问而提问</title>
+  <meta property="og:title" content="Don't ask to ask, just ask" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="zh_CN" />
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
```
Comment (tulir) [inline zh-cn/index.html:9]: I think Open Sans doesn't have non-latin characters, so this should probably be Noto Sans SC. The Korean translation also uses Noto Sans KR.

```suggestion
  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Noto+Sans+SC">
```

File: zh-cn/index.html (hunk 1 above)
Comment (billchenchina) [inline zh-cn/index.html:5]:
```suggestion
  <meta property="og:title" content="不要为了提问而提问，请直接提问" />
```

**Comment by Cyanoxygen** [comment]
👍 Good work! I noticed that two of these reference links are already translated in Chinese, so we can change these links to translated one. :D

**Pull request merged**